target/linux/ramips/dts/mt7620a_wavlink_wl-wn531g3.dts

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7620a.dtsi"
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "wavlink,wl-wn531g3", "ralink,mt7620a-soc";
+	model = "Wavlink WL-WN531G3";
+	
+	aliases {
+		label-mac-device = &wifi0;
```

Comment:
Does label_mac works here? If not, please remove it.

Reply:
This works fine. I get the wifi mac address if i run:
`
root@OpenWrt:~# . /lib/functions/system.sh
root@OpenWrt:~# label_mac_addr=$(get_mac_label)
root@OpenWrt:~# echo $label_mac_addr
`
Is that correct?

Reply:
Then please remove label_mac setting on `02_network` script.

Reply:
I found out that the alias is indeed not working. So I set the label mac in network script. If label mac is not defined it defaults to wifi5g mac. After checking again the label on the device I saw that the correct label mac is the 2g wifi Mac.

Reply:
Everything works correct now